Boost Version 1.74.0

Hello,

Building hipSYCL with Boost/1.74.0 succeeds. My setup is to run celerity on top of hipSYCL. When running applications I get `error while loading shared libraries: libboost_context.so.1.68.0: cannot open shared object file: No such file or directory`. Hoping that I am not doing anything obvious wrong - is Boost/1.74.0 currently supported?

Best regards

Interestingly, I need to use 1.74.0, as I get the following cmake issue when building with 1.78.0:
```
-- Could NOT find Boost (missing: fiber) (found version "1.78.0")
CMake Error at CMakeLists.txt:38 (message):
  Please provide the path to the root directory of the boost installation
  using the -DBOOST_ROOT option
```

I'm on Fedora 35 I am using the boost 1.78.0 .tgz from the website and have it compiled in a directory. I run cmake like this:
```
cmake -G Ninja -DBOOST_ROOT=/home/dan/dep/new_boost/boost_1_78_0 -DCLANG_INCLUDE_PATH=/lib64/clang/13/include ..
```

but get the error:
```
-- Could NOT find Boost (missing: fiber) (found version "1.78.0")
CMake Error at CMakeLists.txt:38 (message):
  Please provide the path to the root directory of the boost installation
  using the -DBOOST_ROOT option
```


When I use a different boost 1.74.0 directory, it works fine. Maybe it's an issue with the FindBoost script being outdated on my platform.

--- a/doc/installing.md
+++ b/doc/installing.md
@@ -10,6 +10,7 @@ In order to successfully build and install hipSYCL, the following dependencies m
 * `cmake`
 * the Boost C++ libraries (in particular `boost.fiber`, `boost.context` and for the unit tests `boost.test`)
   * it may be helpful to set the `BOOST_ROOT` `cmake` variable to the path to the root directory of Boost you wish to use if `cmake` does not find it automatically
+  * **Note for boost 1.78 users:** There seems to be a bug in the build system for boost 1.78, causing the compiled fiber and context libraries not to be copied to the installation directory. You will have to copy these libraries manually to the installation directory. In binary packages from some distribution repositories this issue is fixed. You might be only affected when building boost manually from source.
 
 In addition, the various supported compilation flows have additional requirements (see [here](compilation.md) for more information on available compilation flows):
 

